src/config/dynamic-theme-fixes.config

```diff
@@ -6289,6 +6290,9 @@ a[data-testid] > svg {
 .svelte-15nnlbj {
     font-weight: bold !important;
 }
+#xwd-board rect[class^="Cell-block--"] {
+    fill: var(--darkreader-neutral-background) !important;
```

Comment:
What about `    fill: ${black} !important;`

Reply:
I tried that at first. Based on this note in the readme

>`CSS` rule adds custom CSS to a web page. `!important` keyword should be specified for each CSS property to prevent overrides by other stylesheets. Dynamic mode supports `${COLOR}` template, where `COLOR` is a color value before the inversion (`white` will become `black` in dark mode).

I expected it to invert to white when using the dark scheme. This didn't happen, and the blocked squares are black under both light and dark schemes (same behavior happens if I use `${white}`). I'm not sure if I've just misread that or if there's some issue with the NY Times puzzle that interferes with the extension. I'm fine with black squares for both versions, but it may be low contrast for some folks in dark mode.

Happy to go with whatever approach you prefer here, just wanted to raise that in case the behavior isn't what's expected.

Reply:
> I expected it to invert to white when using the dark scheme. This didn't happen, and the blocked squares are black under both light and dark schemes (same behavior happens if I use ${white}). 

So. Using the `${COLOR}`, only will works in dark mode and will not affect light mode. Your correct it should invert to White, however this fix also inverts the whole board and thus all colors of the cells and makes the white into black.

Reply:
Here's a screenshot showing that the black squares stay black in dark mode using `${black}`.

![image](https://user-images.githubusercontent.com/392720/107864531-20ed3e00-6e2b-11eb-8d5e-1480cc5b93a3.png)

If that's acceptable, works for me. I've just pushed an amended commit with that change.